### DISCUSSION TITLE
Backup DB but exclude the data from certain tables

### DISCUSSION THREAD

**Discussion by de-saurabh** [dbeaver/dbeaver#18683] [DBeaver Development]
With pgdump I can backup a db with all their tables but omit the data of certain tables. The tables will be there but not there data. How do I do this here. I unselected the tables while backing up but after restoring saw that the tables themselves were missing. I do need the tables but not their data. Is this possible to do with the UI?

**Comment by LonwoLonwo** [comment]
Probably, the command "--schema-only" can help. You can add it as an extra argument in the backup settings dialog.
https://www.postgresql.org/docs/current/app-pgdump.html
